comp/core/status/statusimpl/status_test.go

```diff
@@ -421,14 +421,14 @@ X Section
 				expectedStatusHTMLOutput := fmt.Sprintf(`<div class="stat">
   <span class="stat_title">Agent Info</span>
   <span class="stat_data">
-    Version: %s
-    <br>Flavor: %s
-    <br>PID: %d
-    <br>Agent start: 2018-01-05 11:25:15 UTC (1515151515000)
-    <br>Log Level: info
-    <br>Config File: There is no config file
-    <br>Conf.d Path: %s
-    <br>Checks.d Path: %s
+    Version: %s<br>
+    Flavor: %s<br>
+    PID: %d<br>
+    Agent start: 2018-01-05 11:25:15 UTC (1515151515000)<br>
+    Log Level: info<br>
+    Config File: There is no config file<br>
```

Comment:
Should we add a test to so the extra config output?

Reply:
Do you think it would add a lot of added value ? 
I would have to update a big part of the mock config to be able to add extra configuration file in it.
This same test is not even testing `Config File` setting:
>Config File: There is no config file<br>

But if you have a strong opinion on it, I can do it :)